internal/pikamon/util/global.go

```diff
@@ -0,0 +1,16 @@
+package util
+
+import (
+	"github.com/dgraph-io/ristretto"
+)
+
+// Pikamon bot keyword
+var CommandKeyword = "p!ka"
+
+// Create a cache object for the Bot. May contain different caches of varying sizes (used for different purposes)
+type BotCache struct {
+	ChannelCache *ristretto.Cache
+}
+
+// Global metadata variable
+var BotMetadata *BotCache
```

Comment:
This is somewhat of an anti-pattern in Go. Let me dig around and make suggestions on an alternative.

Reply:
Created this issue which I will start working on next
- https://github.com/Jac0bDeal/pikamon/issues/24